ci-operator/step-registry/redhat-chaos/hog-scenarios/io/redhat-chaos-hog-scenarios-io-commands.sh

```
@@ -45,5 +45,9 @@ rc=$?
 if [[ $TELEMETRY_EVENTS_BACKUP == "True" ]]; then
     cp /tmp/events.json ${ARTIFACT_DIR}/events.json
 fi
+
+if [[ $CAPTURE_METRICS == "True" ]]; then 
+  cp $METRICS_PATH ${ARTIFACT_DIR}/metrics-report.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)